sources/src/citygml/externalreference.cpp

```diff
@@ -0,0 +1,19 @@
+//
+//  externalreference.cpp
+//  libcitygml
+//
+//  Created by 松本青空 on 2021/11/15.
+//
+
+#include "citygml/externalreference.h"
```

Comment:
import style

Reply:
Do you mean the following fix?
```suggestion
#import "citygml/externalreference.h"
```

Reply:
no ... even easier  #include <citygml/externalreference.h>

Reply:
OK.